pkg/broker/api_test.go

```diff
@@ -320,34 +329,255 @@ func TestLastOperation(t *testing.T) {
 	assertor.Equal(expected, actual, "should succeed even if stack is not in serviceInstance (was never created)")
 }
 
-func TestBind(t *testing.T) {
-	assertor := assert.New(t)
-
-	opts := Options{
-		TableName:          "testtable",
-		S3Bucket:           "abucket",
-		S3Region:           "us-east-1",
-		S3Key:              "tempates/test",
-		Region:             "us-east-1",
-		BrokerID:           "awsservicebroker",
-		PrescribeOverrides: true,
+func toDescribeStacksOutput(outputs map[string]string) cloudformation.DescribeStacksOutput {
+	var cfnOutputs []*cloudformation.Output
+	for k, v := range outputs {
+		cfnOutputs = append(cfnOutputs, &cloudformation.Output{
+			OutputKey:   aws.String(k),
+			OutputValue: aws.String(v),
+		})
 	}
-	bl, _ := NewAWSBroker(opts, mockGetAwsSession, mockClients, mockGetAccountId, mockUpdateCatalog, mockPollUpdate)
-	bl.db.DataStorePort = mockDataStoreProvision{}
-
-	bindReq := &osb.BindRequest{
-		BindingID:         "test-bind-id",
-		InstanceID:        "exists",
-		AcceptsIncomplete: true,
-		ServiceID:         "test-service-id",
+	return cloudformation.DescribeStacksOutput{
+		Stacks: []*cloudformation.Stack{
+			{
+				Outputs: cfnOutputs,
+			},
+		},
 	}
-	reqContext := &broker.RequestContext{}
+}
 
-	expected := &broker.BindResponse{BindResponse: osb.BindResponse{Credentials: map[string]interface{}{}}}
-	actual, err := bl.Bind(bindReq, reqContext)
-	assertor.Equal(nil, err, "err should be nil")
-	assertor.Equal(expected, actual, "should succeed")
+func TestBind(t *testing.T) {
+	tests := []struct {
```

Comment:
@jaymccon any objections to this style of writing test cases?

Reply:
None. All the tests should look like this :)